elastic/beats
# x-pack/filebeat/input/entityanalytics/provider/{azuread,okta}: add request tracing support

## Proposed commit message



See title.

## Checklist




## Disruptive User Impact



## Author's Checklist



## How to test this PR locally



Run tests in the okta/internal/okta and azuread/fetcher/graph packages with the `-request_trace` flag to see tracing behaviour.

## Related issues


-

## Use cases



## Screenshots



## Logs
